packages/icons/icons.json

```diff
@@ -2,3899 +2,3899 @@
-        "tags": "layers, levels, stack, cards",
+        "tags": "[layers, levels, stack, cards]",
         "group": "interface",
         "codepoint": 61979
     },
     {
         "displayName": "Locate",
         "iconName": "locate",
-        "tags": "target, location, destination, mark, map, area",
+        "tags": "[target, location, destination, mark, map, area, zoom, identify, locate, crosshairs]",
         "group": "action",
         "codepoint": 62001
     },
     {
         "displayName": "Bookmark",
         "iconName": "bookmark",
-        "tags": "bookmark, marker, holder, section, identifier, favorites",
+        "tags": "[bookmark, marker, holder, section, identifier, favorites, tag, save]",
         "group": "action",
         "codepoint": 61751
     },
     {
         "displayName": "Citation",
         "iconName": "citation",
-        "tags": "quotation, citation, marks, excerpt",
+        "tags": "[quotation, citation, marks, excerpt, quote, end quote, grammar, punctuation]",
         "group": "editor",
         "codepoint": 61780
     },
     {
         "displayName": "Tag",
         "iconName": "tag",
-        "tags": "tag, label, badge, identification",
+        "tags": "[tag, label, badge, identification]",
         "group": "action",
         "codepoint": 62175
     },
     {
         "displayName": "Clipboard",
         "iconName": "clipboard",
-        "tags": "clipboard, notepad, notebook, copy, paste, transfer, storage",
+        "tags": "[clipboard, notepad, notebook, copy, paste, transfer, storage, note]",
         "group": "action",
         "codepoint": 61783
     },
     {
         "displayName": "Selection",
         "iconName": "selection",
-        "tags": "selection, collection, circle, ring",
+        "tags": "[selection, collection, circle, ring]",
         "group": "action",
         "codepoint": 62117
     },
     {
         "displayName": "Events",
         "iconName": "timeline-events",
-        "tags": "calendar, timeframe, agenda, diary, day, week, month",
+        "tags": "[calendar, timeframe, agenda, diary, day, week, month, schedule]",
         "group": "interface",
         "codepoint": 62195
     },
     {
         "displayName": "Line chart",
         "iconName": "timeline-line-chart",
-        "tags": "graph, line, chart",
+        "tags": "[graph, line, chart, line chart, upward trend, time series, line graph]",
         "group": "data",
         "codepoint": 62196
     },
     {
         "displayName": "Bar chart",
         "iconName": "timeline-bar-chart",
-        "tags": "graph, bar, chart",
+        "tags": "[graph, bar, chart, horizontal histogram, bar chart]",
         "group": "data",
         "codepoint": 62194
     },
     {
         "displayName": "Applications",
         "iconName": "applications",
-        "tags": "application, browser, windows, platforms",
+        "tags": "[application, browser, windows, platforms, pages, duplicate, copy, page, window, website, files]",
```

Comment:
I don't know if I agree. The reason I crowdsourced this icon tagging was because people have a hard time finding icons. I think accepting people's loose associations will make searching a lot more powerful. If this icon comes up when someone searched "files" or "pages" I don't think it would be an issue. They'll have different options and they can choose one. But if you disagree I'm happy to remove them!

Reply:
let's keep "page" (with the intention that it's an abbreviation of "webpage"), add "webpage" too, and leave out "files". kind of a middle ground.

Reply:
done!